### PR TITLE
Updating DELETE responses codes

### PR DESCRIPTION
**Pull request by pasuder**
Due to RFC2616, section 9.7. See [this section](http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html).

### CODE REVIEW

**Comment by pasuder** [comment]
@nowakowski please check it if does suit your requirements.
@mkasztelnik please check it if Atmospheres DNAT wrangler client will works after this change.

**Comment by mkasztelnik** [comment]
There are a lot of repetitions in this code. Can you extract it into one method and reuse it when possible?

**Comment by pasuder** [comment]
@mkasztelnik yes, I extract recurrent code to method. I am working on transforming `ip-wrangler` to gem in branch [`task_create_gem`](https://github.com/dice-cyfronet/ip-wrangler/tree/task_create_gem).

**Comment by mkasztelnik** [comment]
Ok. We can postpone my remark till it is merged into gem. I will create appropriate issue describing this problem.

:+1:

For sure introduced changes will brake atmosphere. We need to coordinate deployment of new ipwrangler on dev and prod with appropriate changes in atmosphere.

**Comment by mkasztelnik** [comment]
Mentioned issue created and can be found here #8

**Comment by pasuder** [comment]
Removed repetitions in edited code. @mkasztelnik would like to check it? I will use this changes in branch `task_create_gem`.

**Comment by mkasztelnik** [comment]
Also this line is repeated all the time:

```
released_port = $nat.release_port private_ip, private_port, protocol
```

Maybe it can also be added into `check_released_resource` (then for sure name need to be changed)?

**Comment by mkasztelnik** [comment]
:+1:

**Comment by pasuder** [comment]
Buah! I will use this branch in `task_create_gem`

**Comment by pasuder** [comment]
@dice-cyfronet/atmo-dev-team would like to review this PR?

**Comment by mkasztelnik** [comment]
:+1:

**Comment by pasuder** [comment]
@mkasztelnik yours second :+1: does not work again :smiley:  Still only 1 thumb up.

**Comment by mkasztelnik** [comment]
I know, it was a motivation for the others :smile_cat:

File: src/ipt_main.rb [hunk 1]
```diff
@@ -40,6 +40,24 @@ def valid_protocol?(protocol)
   protocol =~ /^(tcp|udp)$/i ? true : false
 end
 
+def release_ip_and_check(private_ip, public_ip = nil)
+  released_ip = $nat.release_ip private_ip, public_ip
```
Comment (tbartynski) [inline src/ipt_main.rb:44]: Method invocation without braces makes the code difficult tot read.

**Comment by tbartynski** [comment]
:+1: 
but I don't like that you don't use braces to indicate method invocations.

File: src/ipt_main.rb (hunk 1 above)
Comment (pasuder) [inline src/ipt_main.rb:44]: That's true. I would like to changes this in branch [`task_create_gem`](https://github.com/dice-cyfronet/ip-wrangler/tree/task_create_gem). See PR #4

**Comment by pasuder** [comment]
I leave this in that version (without braces) to being consequent.

**Pull request merged**